code/game/gamemodes/changeling/powers/headcrab.dm

```diff
@@ -7,34 +7,37 @@
 	req_human = 1
 
 /obj/effect/proc_holder/changeling/headcrab/sting_action(mob/user)
-	var/datum/mind/M = user.mind
-	var/list/organs = user.get_organs_zone("head", 1)
+	if(alert("Are you sure you wish to do this? This action cannot be undone.",,"Yes","No")=="No")
+		return
+	else
```

Comment:
I don't think this `else` block is necessary, due to you `return`ing immediately after checking for `No`, and not checking anything else

Reply:
Thank you, I think I fixed what you're talking about!